server/src/main/java/org/elasticsearch/script/ReadDocValuesField.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.script;
+
+import org.elasticsearch.index.fielddata.ScriptDocValues;
+
+import java.util.List;
+
+public class ReadDocValuesField<T> implements Field<T> {
```

Comment:
On further though, I would just remove the "read" part of this. Doc values are by definition read only.

Reply:
Removed